summingbird-online/src/main/scala/com/twitter/summingbird/planner/OnlinePlan.scala

```diff
@@ -27,6 +29,17 @@ class OnlinePlan[P <: Platform[P], V](tail: Producer[P, V]) {
   private val depData = Dependants(tail)
   private val forkedNodes = depData.nodes
     .filter(depData.fanOut(_).exists(_ > 1)).toSet
+
+  private def get[T <: AnyRef: ClassTag](dep: Producer[P, _]) = {
+    Options.getFirst[T](options, nameMap.get(dep).get)
+  }
+
+  private def getOrElse[T <: AnyRef: ClassTag](dep: Producer[P, _], default: T): T =
+    get[T](dep) match {
```

Comment:
Minor, could be written more succinctly as 
get[T](dep).map { case (namedSource, option) => option }.getOrElse(default)